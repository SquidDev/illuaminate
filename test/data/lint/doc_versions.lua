-- config: (lint (only doc:unordered-versions))

--- A module
--
-- @since 1.0
-- @changed 1.1
-- @since 1.2
