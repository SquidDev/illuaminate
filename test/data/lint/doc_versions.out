doc_versions.lua:[6:4-6:11]: @since tag for version "1.1" must be the first changelog entry. [doc:unordered-versions]
   │
 6 │ -- @changed 1.1
   │    ^^^^^^^^
1 errors and 0 warnings
